src/drag-and-drop.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable  no-restricted-syntax, no-loop-func */
+
+import { checkboxesUpdate } from './status-updates.js'; // eslint-disable-line
+import { setStorage } from './index.js'; // eslint-disable-line
+import { ToDoList } from './arr-crud'; // eslint-disable-line
+import { toDoListArr } from './index.js'; // eslint-disable-line
+
+const showTasks = document.querySelector('#tasks');
+
+function renderTasks(tasksArr) {
+  showTasks.innerHTML = '';
+  tasksArr.forEach((task) => {
+    let checked = '';
+    if (task.completed) {
+      checked = 'checked';
+    }
+    showTasks.innerHTML += `<li draggable="true" class="task pos-${task.index}" data-index="${task.index}"><div class="task-to-do"><input class="checkbox" data-index="${task.index}" type="checkbox" ${checked}> ${task.description}</div><button class="btn-more" data-index="${task.index}">more<i class="fas fa-info-circle"></i></button></li>`;
+  });
+}
+
+function arraymove(arr, fromIndex, toIndex) {
+  const element = arr[fromIndex];
+  arr.splice(fromIndex, 1);
+  arr.splice(toIndex, 0, element);
+}
+
+function dragndrop(oriArr) {
```

Comment:
Follow [good naming practices](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) the function name and the parameter name don't mean anything neither express intent or functionality.
A better name would be: dfineDragBehivour, or makeListElementDraggable. I kindly suggest you take a look at the link I provided.